### ISSUE TITLE
[BUG] Website is dead for 24+ hours

### ISSUE THREAD

**Issue by NinoDLC** [valheimPlus/ValheimPlus#794]
http://valheim.plus/ is blank

**Comment by GraveofBears** [comment]
lol 24+ hours? thing has been down for a year...

**Comment by kmayberry91** [comment]
and the discord is full of sheeple saying to read pinned posts on this dead project

**Comment by Fenrirthviti** [comment]
Duplicate of https://github.com/valheimPlus/ValheimPlus/issues/723

**Comment by anon2828226** [comment]
which is probably a duplicate of #698 and #730. also nx said that would be fixed "in a few days".

**Comment by Agocelt** [comment]
You can still access the site via web archive:
https://web.archive.org/web/20220608102318/https://valheim.plus/documentation/

**Comment by NinoDLC** [comment]
Yes sorry I'm new to Valheim so I didn't want to make any asumption. And for discord, the link is dead too so I couldn't join the discord either...

**Comment by BazkieBumpercar** [comment]
If the devs don't have time to support this mod anymore, maybe it's an idea for them to transfer the project to volunteers that want to take over (assuming there's people available)?

It'd be a shame for this to die; I don't really want to play Valheim anymore without these additions, but I don't want to miss out on Valheim patches either.. and I'm sure I'm not the only one! (I mean, this is like the most popular big mod on nexusmods' Valheim section, so it'd be just weird if it died!)

**Comment by PuszkaPotato** [comment]
From what I know, the main developer is MIA but just join the [discord server](https://discord.com/invite/XamVGpgnJT) or follow updates on [Thunderstore ](https://valheim.thunderstore.io/package/Grantapher/ValheimPlus_Grantapher_Temporary/) or [Nexus](https://www.nexusmods.com/valheim/mods/2323?tab=description).

Just so you know, the files on Thunderstore are from Grantapher and the files on Nexus are from me. It doesn't really matter which you choose but Grantapher definitely has more experience with Valheim modding than I do.

**Comment by BazkieBumpercar** [comment]
Thanks, that's great!

**Comment by JF10R** [comment]
> If the devs don't have time to support this mod anymore, maybe it's an idea for them to transfer the project to volunteers that want to take over (assuming there's people available)?
> 
> It'd be a shame for this to die; I don't really want to play Valheim anymore without these additions, but I don't want to miss out on Valheim patches either.. and I'm sure I'm not the only one! (I mean, this is like the most popular big mod on nexusmods' Valheim section, so it'd be just weird if it died!)

I doubt it would die. Being open-source, we can keep improving on it and making changes along the patch. For example, in the last two days I've added the Wisp Spawner and the Eitr Refinery to my own version of ValheimPlus. I might release it soon, but I'm still working on more stuff :)

**Comment by ersan** [comment]
> I doubt it would die. Being open-source, we can keep improving on it and making changes along the patch. For example, in the last two days I've added the Wisp Spawner and the Eitr Refinery to my own version of ValheimPlus. I might release it soon, but I'm still working on more stuff :)

It might be better if you collaborate with Grantapher and come up with one repo to replace this one, splintering into a bunch of different forks doesn't seem like a great idea.

**Comment by JF10R** [comment]
@ersan 
I'm doing pull requests on the official repo for the moment (#797 #798). 

About Grantapher's fork, I closed my pull request over there because I don't want to _force_ them into a "new ValheimPlus Manager" position if they don't feel like managing everything that goes with it. I'm all ears if you have suggestions on how to proceed though!

 I'm definitely with you on the fact that a single repo/fork should be considered the new "main" repo, should @nxPublic never come back.

**Comment by IMBeniamin** [comment]
I have been hosting at this point probably for about a year the official website, unofficially [here](https://imben.alwaysdata.net/) because it contains a huge amount of very useful information.
This is a copy I got from the wayback machine so it might be a bit outdated, but it should still be mostly fine.

**Comment by kblaschke** [comment]
Also as a word of warning, the official URL now redirects to some shady advertising websites, so better  don't click on that link anymore. As it happens, you might even be offered downloads which make you think it's ValheimPlus, but instead you've now got a bunch of malware.

**Comment by kachunkachunk** [comment]
Echoing this warning. Someone's squatting on the domain and I doubt it would be worthwhile trying to purchase it any time soon (due to squatters putting crazy costs on them), should that be on mind.

This also means that all references to the site via documentation need to be updated/changed, and so do all the config file comments.

**Comment by kblaschke** [comment]
uBlock Origin now blocks requests to this domain due to "badware risks". If you don't have an adblocker or other means of browser protection, **I strongly recommend to not visit valheim.plus at all!**

**Comment by IMBeniamin** [comment]
> I have been hosting at this point probably for about a year the official website, unofficially [here](https://imben.alwaysdata.net/) because it contains a huge amount of very useful information. This is a copy I got from the wayback machine so it might be a bit outdated, but it should still be mostly fine.

The website copy is still up at this address and I will keep it up as long as possible.
I have tried contacting the developer multiple times without success due to various reasons, I have also spoken with the mod team in the discord and I think the best way to proceed would be to make this a community effort.

If someone is interested in contributing please contact me so I can take the necessary steps to allow community modifications to the website.
I have noticed also other efforts by the community to support this plugin, so if anyone is interested in some form of integration, again, please leave a message here.

A note:
The original website was being dynamically created from a json file, sadly re-implementing this would require quite a bit of work, but is doable.
